<launch>

   <node pkg="hratc2016_entry_template" type="random_walk_navigation_node" respawn="false" name="random_walk" output="screen">
		<param name="min_x" value="-10.0"/>
		<param name="max_x" value="10.0"/>
		<param name="min_y" value="-10.0"/>
		<param name="max_y" value="10.0"/>
  	</node>

</launch>
